src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanel.js

```diff
@@ -66,15 +67,12 @@ class DocumentPanel extends Component {
                         metadata={doc.metadata}
                       />
                       {doc.metadata.imprint?.publisher && (
-                        <>
-                          <div className="default-margin-bottom">
-                            Published by{' '}
-                            <b>{doc.metadata.imprint?.publisher}</b>{' '}
-                            {doc.metadata.publication_year && (
-                              <b>{doc.metadata.publication_year}</b>
-                            )}
-                          </div>
-                        </>
+                        <div className="default-margin-bottom">
+                          Published by <b>{doc.metadata.imprint?.publisher}</b>{' '}
+                          {doc.metadata.publication_year && (
```

Comment:
minor: if I remember correctly publication_year is required record property so probably this check is redundant

Reply:
none of the publications fields are required from what i can see!

Reply:
publication year is not part of the imprint but it is on the parent level. Did you check the json schema required fields?

Reply:
ah my bad, you are right